### PR TITLE
fix edit links.. update Wix docs directory

### LINKED ISSUE #1443: Link to edit Wix page is leading to wrong page
It is trying to navigate to https://github.com/btcpayserver/wix.git/edit/master/docs//Wix.md, see video:

https://github.com/user-attachments/assets/a0845e4e-9859-4be8-9ec7-2b420ed1130d

This issue seems to be with all the docs that have setup as external dependency, like in this PR https://github.com/btcpayserver/btcpayserver-doc/pull/1435

cc @pavlenex @TChukwuleta

### PR DESCRIPTION
**Pull request by TChukwuleta**
Resolves #1443 

Here @dennisreimann, kindly review

### CODE REVIEW

**Review by dennisreimann — APPROVED** [review]
There were some more changes necessary. I also simplified the update external bash function. Can you also give this a quick review again?

File: setup-deps.sh [hunk 1]
```diff
@@ -330,15 +325,14 @@ fi
...
 done
 
 # Wix
 echo "Setup dependency: Wix"
 
-if [ -f "$DOCS_DIR/Wix.md" ]; then
-  rm "$DOCS_DIR/Wix.md"
-fi
+rm -rf "$DOCS_DIR/Wix"
+mkdir -p "$DOCS_DIR/Wix"
```
Comment (TChukwuleta) [inline setup-deps.sh:335]: Hey @dennisreimann  there is a duplicate command here

**Comment by TChukwuleta** [comment]
Tested them again.. all good

File: setup-deps.sh (hunk 1 above)
Comment (rockstardev) [reply setup-deps.sh:335]: @TChukwuleta @dennisreimann pushed commit to resolve this 9d5abd62713367c975b19b28dc22f43cd17fde4b

Comment (dennisreimann) [reply setup-deps.sh:335]: Thanks for spotting and fixing!

**Pull request merged**